coverage/com.example.campusguide.utils/index.html

```diff
@@ -0,0 +1,125 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
Do we want to keep the coverage report under version control?

Reply:
Nope! Removing